dist/configs/itwinjs-recommended.js

```diff
@@ -228,7 +228,7 @@ module.exports = {
     "@typescript-eslint/unbound-method": [
       "error",
       {
-        "ignoreStatic": true
+        "ignorestatic": false
```

Comment:
```suggestion
        "ignoreStatic": false
```

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/unbound-method.md#ignorestatic

Reply:
I hate laptop keyboards.

Reply:
Do you want me to submit another PR?

Reply:
dw, ill do it